src/Routes/routes.js

```diff
@@ -19,7 +20,11 @@ function routes(app) {
 
     app.get('/api/user/:id', jsonParser, UserController.getUserById, (req, res) => {
         res.send('Here is a specific user')
-    })
+    });
+
+    app.get('/api/track/:track', jsonParser, TrackController.getTrackResults, (req, res) => {
```

Comment:
Should it not be api/tracks/:track?

Reply:
RESTful api says it should be singular, according to Mike